ci/install-3.5_NUMPY_DEV.sh

```diff
@@ -12,7 +12,8 @@ pip uninstall numpy -y
 # these wheels don't play nice with the conda libgfortran / openblas
 # time conda install -n pandas libgfortran openblas || exit 1
 
-time sudo apt-get $APT_ARGS install libatlas-base-dev gfortran
+# Not going to work in containers
```

Comment:
Should I invent or find a way if doing something like:
```
if in container:
    sudo install package
```
or is it better to just remove all sudos from the travis scripts?